commercetools-models/src/main/java/io/sphere/sdk/shoppinglists/LineItemDraftBuilder.java

```diff
@@ -25,4 +27,26 @@
     public LineItemDraftBuilder custom(@Nullable final CustomFields custom) {
         return super.custom(CustomFieldsDraftBuilder.of(custom).build());
     }
+
+    /**
+     * Creates a new object initialized with the given values.
+     *
+     * @param variantIdentifier initial value for the {@link io.sphere.sdk.carts.LineItemDraft#getProductId()} and {@link io.sphere.sdk.carts.LineItemDraft#getVariantId()} properties
+     * @param quantity          initial value for the {@link io.sphere.sdk.carts.LineItemDraft#getQuantity()} property
+     * @return new object initialized with the given values
+     */
+    public static LineItemDraftBuilder ofVariantIdentifier(final ByIdVariantIdentifier variantIdentifier, @Nullable final Long quantity) {
```

Comment:
Does it make sense to add the quantity attribute here?

Reply:
This also follows the pattern of `carts/LineItemDraftBuilder`.

Reply:
Cool 😄  now i see its a consistency issue.
Thanks for the explanation 👍